improver_tests/utilities/test_cube_checker.py

```diff
@@ -33,28 +33,33 @@
 import unittest
 
 import iris
+import numpy as np
 from iris.cube import Cube
 from iris.exceptions import CoordinateNotFoundError
 from iris.tests import IrisTest
 
+from improver.synthetic_data.set_up_test_cubes import (
+    add_coordinate,
+    set_up_variable_cube,
+)
 from improver.utilities.cube_checker import (
     check_cube_coordinates,
     check_for_x_and_y_axes,
     find_dimension_coordinate_mismatch,
     spatial_coords_match,
 )
 
-from ..nbhood.nbhood.test_BaseNeighbourhoodProcessing import set_up_cube
-
 
 class Test_check_for_x_and_y_axes(IrisTest):
 
     """Test whether the cube has an x and y axis."""
 
     def setUp(self):
         """Set up a cube."""
-        self.cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2),), num_time_points=1, num_grid_points=5
+        data = np.ones((1, 5, 5), dtype=np.float32)
+        data[:, 2, 2] = 0.0
```

Comment:
Are these data values ever used?  If not, we don't need to set this point.

Reply:
Removed.